packages/react-ai/package.json

```diff
@@ -42,7 +42,8 @@
     "typecheck": "tsc --noEmit"
   },
   "peerDependencies": {
-    "@aws-amplify/api-graphql": "unstable",
+    "@aws-amplify/api-graphql": "^4.7.8",
+    "@aws-amplify/data-schema": "^1.19.0",
```

Comment:
I don't know why we had a peerDep on unstable for api-graphql package, probably an oversight on my part when going to GA. I just grabbed the latest current version of that package (4.7.8). We need have a dep on this package to make eslint happy because we use a type definition in our types. But this change needs the latest release of @aws-amplify/data-schema, which @aws-amplify/api-graphql depends on, but hasn't updated its version yet. This probably warrants a good discussion around.